### PR TITLE
- Rename module location

### PR DESCRIPTION
**Pull request by rjschwei**
+ Rename the Python package from ec2utils to ec2imgutils to conform to
    established convention
  + Fix the import statement
  + File renaming to reflect overall name changes

### CODE REVIEW

File: ec2deprecateimg [hunk 1]
```diff
@@ -22,9 +22,9 @@ import argparse
 import os
```
Comment (smarlowucf) [inline ec2deprecateimg]: Consider updating copyright for consistency with other modules.

File: ec2publishimg [hunk 2]
```diff
@@ -21,9 +21,9 @@ import argparse
 import os
```
Comment (smarlowucf) [inline ec2publishimg]: Ditto

File: ec2uploadimg [hunk 3]
```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/python3
```
Comment (smarlowucf) [inline ec2uploadimg]: Ditto

**Comment by rjschwei** [comment]
Good catch thanks, fixed.

File: ec2deprecateimg [hunk 4]
```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/python
 
-# Copyright 2015 SUSE LLC, Robert Schweikert
+# Copyright 2015 SUSE LLC
 #
 # This file is part of ec2deprecateimg.
```
Comment (smarlowucf) [inline ec2deprecateimg]: Aren't the scripts part of ec2imgutils?

**Comment by rjschwei** [comment]
I think I got everything in lib, obviously missed the scripts. All fixed now.

**Pull request merged**